server/src/service/impl/UserAvatarServiceDicebear.ts

```diff
@@ -0,0 +1,37 @@
+import axios, { AxiosInstance } from "axios";
+import { AbstractUserAvatarService } from "../AbstractUserAvatarService";
+
+
+class UserAvatarServiceDicebear extends AbstractUserAvatarService {
+
+  private static INSTANCE: UserAvatarServiceDicebear;
+
+  private axiosInstance: AxiosInstance;
```

Comment:
**chore:**
1. Remove linha em branco no início da classe
2. Remover espaço em branco entre propriedades
3. As injeções `axiosInstance` e `INSTANCE` não deveria ser parâmetro do construtor?
4. o nome INSTANCE não é claro quanto ao significado do que se refere a injeção, recomendo algo como : `avatarService` ou `dicebearAvatarService`

Reply:
3 - O axiosInstance  pode ser sim. Já a INSTANCE, como essa classe segue o padrão singleton, ela deve ser instanciada na própria classe.